k2/csrc/determinize.cc

```diff
@@ -68,15 +66,18 @@ struct DetStateElement {
 struct DetState {
   // `base_state` is a state in the input FSA.
   int32_t base_state;
-  // seq_len is the length of symbol sequence that we follow from state `base_state`.
-  // The sequence of symbols can be found by tracing back one of the DetStateElements
-  // in the doubly linked list (it doesn't matter which you pick, the result will be the
+  // seq_len is the length of symbol sequence that we follow from state
+  // `base_state`.
+  // The sequence of symbols can be found by tracing back one of the
+  // DetStateElements
+  // in the doubly linked list (it doesn't matter which you pick, the result
+  // will be the
   // same.
   int32_t seq_len;
 
-  bool normalized { false };
+  bool normalized{false};
```

Comment:
For consistency, other members should also be initialized.

Reply:
Please donot comments on this file as it it even not compiled. I just fix style issues to pass the style check.